cognite_toolkit/_cdf_tk/load/_base_loaders.py

```diff
@@ -243,34 +222,21 @@ def dump_resource(
         """
         return resource.dump(), {}
 
+    @abstractmethod
     def create(self, items: T_CogniteResourceList) -> Sized:
-        return self.api_class.create(items)
+        raise NotImplementedError()
 
+    @abstractmethod
     def retrieve(self, ids: SequenceNotStr[T_ID]) -> T_WritableCogniteResourceList:
-        if hasattr(self.api_class, "retrieve_multiple"):
-            if inspect.signature(self.api_class.retrieve_multiple).parameters.get("ignore_unknown_ids"):
-                retrieved = self.api_class.retrieve_multiple(external_ids=ids, ignore_unknown_ids=True)
-            else:
-                retrieved = self.api_class.retrieve_multiple(external_ids=ids)
-        else:
-            retrieved = self.api_class.retrieve(ids)
-        if retrieved is None:
-            return self.list_cls([])
-        elif isinstance(retrieved, self.list_cls):
-            return retrieved
-        elif isinstance(retrieved, Collection):
-            return self.list_cls(retrieved)
-        else:
-            return self.list_cls([retrieved])
+        raise NotImplementedError()
 
+    @abstractmethod
     def update(self, items: T_CogniteResourceList) -> Sized:
-        return self.api_class.update(items)
+        raise NotImplementedError()

```

Comment:
How about something like this, as a reminder? Or is it obvious when it is @abstractmethod?

```suggestion
        raise NotImplementedError("update must be implemented in all Resource Loader classes")

```

Reply:
It should be obvious, https://stackoverflow.com/questions/40894284/what-should-i-put-in-the-body-of-an-abstract-method

Raising a NotImplementedError in abstractmethod means it should absolutely be implemented.